### ISSUE TITLE
Homepage accessibility audit

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3961]
### Descriptive summary

The following six points are actions that need to be taken on the Hyrax homepage in order to be more accessible: 

1. 
The `#featureTab` and `#recentTab` role should be contained by a required ARIA parents role such as "tablist". 

![Screen Shot 2019-09-11 at 3 34 22 PM](https://user-images.githubusercontent.com/9621070/64740274-abe53e80-d4a9-11e9-9836-70a86aab8f8b.png)

2. 
`<li class="active"><a aria-expanded="true" href="#tab-col2-first" role="tab" data-toggle="tab">Explore Collections</a></li>` does not have a `<ul>, <ol> or role="list" `parent element

![Screen Shot 2019-09-11 at 3 35 49 PM](https://user-images.githubusercontent.com/9621070/64740336-e3ec8180-d4a9-11e9-8c02-5a298163a80e.png)

3. 
`<li class=""><a aria-expanded="false" href="#tab-col2-second" role="tab" data-toggle="tab">Featured Researcher</a></li>` does not have a `<ul>, <ol> or role="list"` parent element

![Screen Shot 2019-09-11 at 3 38 48 PM](https://user-images.githubusercontent.com/9621070/64740452-45aceb80-d4aa-11e9-8988-725ba3aabcbc.png)

4. 
`#masthead` must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

![Screen Shot 2019-09-11 at 3 40 30 PM](https://user-images.githubusercontent.com/9621070/64740593-b0f6bd80-d4aa-11e9-8022-8e1071bfc848.png)

5. 
`<html lang="en" prefix="og:http://ogp.me/ns#" class="fixedsticky-withoutfixedfixed"> ` Page must have a level-one heading

![Screen Shot 2019-09-11 at 3 42 34 PM](https://user-images.githubusercontent.com/9621070/64740630-d1267c80-d4aa-11e9-9eec-391e994c6e44.png)

6. 
`<html lang="en" prefix="og:http://ogp.me/ns#" class="fixedsticky-withoutfixedfixed">`  is not contained by landmarks 

![Screen Shot 2019-09-11 at 3 43 38 PM](https://user-images.githubusercontent.com/9621070/64740671-f5825900-d4aa-11e9-97f2-84e4d9983270.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/ 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab 
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, unable to tab through the links and buttons. 

Chrome works well. 

![Screen Recording 2019-09-16 at 01 42 PM](https://user-images.githubusercontent.com/9621070/64992075-00146800-d888-11e9-89b7-5fa113d99054.gif)

**Comment by dlim87** [comment]
## Tab test
selects all the elements in firefox, chrome, safari, and edge

## Axe audit
Has 4 problems:
### All page content must be contained by landmarks (1)
![Image 2019-11-06 at 10 17 28 AM](https://user-images.githubusercontent.com/5492162/68325621-a443af80-007e-11ea-9857-d000ab64a56e.png)
### Elements must have sufficient color contrast (3)
![Image 2019-11-06 at 10 18 26 AM](https://user-images.githubusercontent.com/5492162/68325687-c89f8c00-007e-11ea-80ba-2873d33dfecc.png)
![Image 2019-11-06 at 10 18 46 AM](https://user-images.githubusercontent.com/5492162/68325715-d523e480-007e-11ea-8607-92be96a41e8e.png)
![Image 2019-11-06 at 10 19 06 AM](https://user-images.githubusercontent.com/5492162/68325746-e10fa680-007e-11ea-97e8-eb79c2e9eb6c.png)

**Comment by Dananji** [comment]
@dlim87 When I run this in my local dev environment, I don't get the first issue you have mentioned: `All page content must be contained by landmarks (1)`.

All the related nodes in this issue are the elements introduced in the Nurax deployment, which are not in the Hyrax code. 
Maybe we can say this is fixed because it is not in the Hyrax code ??

**Comment by no-reply** [comment]
@Dananji can we identify *how* they are introduced in the Nurax code (e.g. a link to the relevant lines in the Nurax repository)?

if the a11y issues are very specifically isolated to Nurax, then i think we can close this. if there are changes we can make on the Hyrax side to improve the situation for downstream apps, then we should identify them for this.

there's not much Nurax-specific code, so i'd like to identify the root of this problem before moving on.

**Comment by dlim87** [comment]
Upon further examination, I think it's the content blocks that are triggering it.

When first booting up a local instance, I only get the color errors, but when I add some marketing text (for the banner text), I get a very similar accessibility complaint.

Also, when I have an announcement text content block, I get a complaint about an id being used twice: 
![Image 2019-11-07 at 1 09 08 PM](https://user-images.githubusercontent.com/5492162/68427888-da0c9500-015f-11ea-874f-efccaf40d3b5.png)
![Image 2019-11-07 at 1 09 02 PM](https://user-images.githubusercontent.com/5492162/68427893-dc6eef00-015f-11ea-8028-91dbe742c2e4.png)

**Comment by Dananji** [comment]
@no-reply :arrow_up:
@dlim87 I think you are right. This could be the root of this issue. I didn't know that's how these nodes were introduced into the website. Thanks!
I will look into this :+1:

**Comment by MariaStarPower** [comment]
Results of the analysis for the **Homepage**:

<details>

![Image 2020-09-10 at 2 22 52 PM](https://user-images.githubusercontent.com/32848057/92809102-296bb500-f371-11ea-9b85-287aa7f5365e.png)

<img width="1428" alt="Image 2020-09-10 at 2 23 49 PM" src="https://user-images.githubusercontent.com/32848057/92809241-4a340a80-f371-11ea-85d9-01a52cf7b4e3.png">

<img width="1426" alt="Image 2020-09-10 at 2 24 39 PM" src="https://user-images.githubusercontent.com/32848057/92809417-73549b00-f371-11ea-8c15-24020eeccec2.png">

<img width="1427" alt="Image 2020-09-10 at 2 25 46 PM" src="https://user-images.githubusercontent.com/32848057/92809515-8cf5e280-f371-11ea-9743-676626551e3e.png">

</details>

**Comment by MariaStarPower** [comment]
I am able to tab through the links and buttons on the page in Chrome and Firefox, but not in Safari.  I have found that the Chrome browser is the best for tabbing.

**Comment by jlhardes** [comment]
This issue seems to be a report of an accessibility audit and might point to multiple things that need to be fixed. Review needed to determine work that remains to be done. Not a blocker for Hyrax 3.0 release.

**Comment by jlhardes** [comment]
I am able to tab through the home page on nurax-dev in Safari (with the following System Preference set on the Mac: Full Keyboard Access set to All Controls in System Preferences > Keyboard > Shortcuts). This issue is closed and done.
